var/spack/repos/builtin/packages/opencv/package.py

```diff
@@ -68,7 +68,7 @@ class Opencv(CMakePackage):
     variant('java', default=False,
             description='Activates support for Java')
     variant('openmp', default=False, description='Activates support for OpenMP threads')
-    variant('core', default=False, description='Include opencv_core module into the OpenCV build')
+    variant('core', default=True, description='Include opencv_core module into the OpenCV build')
```

Comment:
I'm curious why you changed the default to `True` here - do you think most users would want this enabled? Or is it not building for you when `core=False`?